<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationMonitor</class>
 <widget class="QMainWindow" name="SimulationMonitor">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>170</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>170</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>102</green>
        <blue>100</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>alien monitor</string>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>190</height>
    </size>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>number of clusters:</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberClustersLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>9</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>24</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>number of cells:</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberCellsLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>24</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>39</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>number of energy particles:</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberEnergyParticlesLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>39</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>53</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>number of token:</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberTokenLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>53</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>68</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>total internal energy:</string>
    </property>
   </widget>
   <widget class="QLabel" name="internalEnergyLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>68</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>82</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>total kinetic energy:</string>
    </property>
   </widget>
   <widget class="QLabel" name="kinEnergyLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>82</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>97</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> - translational parts:</string>
    </property>
   </widget>
   <widget class="QLabel" name="transEnergyLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>97</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rotEnergyLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>111</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>126</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>overall energy:</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalEnergyLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>126</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>111</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> - rotational parts:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
